Tools/peg_generator/pegen/c_generator.py

```diff
@@ -32,7 +32,7 @@
 #include "pegen.h"
 
 #if defined(Py_DEBUG) && defined(Py_BUILD_CORE)
-#  define D(x) if (Py_DebugFlag) x;
+#  define D(x) if (_Py_GetConfig()->parser_debug) { x; }
```

Comment:
I wonder if this makes the debug run slower due to the function call (is everywhere)🤔

Could you do a quick check please?

Reply:
This change only affects the debug build. I don't know how to measure the parser performance. Any suggestion?

Reply:
I know this only affects debug mode, but I want to know by how much.

A quick check is running with "python -d" some big python file that doesn't actually execute any code and compare the timing before and after this PR. I don't need anything exact, just a rough estimate